1.7/administration/installing/custom/create-public-agent.md

```diff
@@ -4,69 +4,64 @@ nav_title: Public Agent
            0
-    ```
+    
 
 ### Prerequisites:
 
-1. DC/OS is installed and you have deployed at least one private agent node.
+*   DC/OS is installed and you have deployed at least one private agent node.
+*   These steps must be performed on a machine that is configured as a DC/OS node. 
+*   Any tasks that are running on the node will be terminated during this conversion process.
 
-### Converting private agents to public:
-
-**The following steps must be taken on a machine that is currently a DC/OS node. If this node is currently running tasks, those tasks will be terminated in this process.**
+### Uninstall the DC/OS private agent software
 
 1.  Uninstall the current DC/OS software on the agent node.
-
-    ```bash
-    $ sudo -i /opt/mesosphere/bin/pkgpanda uninstall
-    $ sudo systemctl stop dcos-mesos-slave
-    $ sudo systemctl disable dcos-mesos-slave
-    ```
-
-1.  Remove the old directory structures on the agent node.
-
-    ```bash
-    $ sudo rm -rf /opt/mesosphere /var/lib/mesos
-    ```
-
-1.  Restart the machine.
-
-    ```bash
-    $sudo reboot
-    ```
-
-### Install DC/OS
-
-1.  Copy the `dcos-install.tar` file created in the GUI or CLI installation method to the machine that will be a public agent.
-
-    ```bash
-    $ ssh $USER@$AGENT "sudo mkdir -p /opt/dcos_install_tmp"
-    $ scp ~/dcos-install.tar $ROOT_USER@$AGENT:~/dcos-install.tar
-    ```
-
-1.  SSH to the machine:
-
-    ```bash
-    $ ssh $USER@$AGENT
-    ```
-
-1.  Install DC/OS as a public agent:
-
-    ```bash
-    $ sudo mv ~/dcos-install.tar /opt/dcos_install_tmp
-    $ sudo bash /opt/dcos_install_tmp/dcos_install.sh slave_public
-    ```
-
-1.  Verify that your new agent node is public by running this command from DC/OS CLI.
-
-    ```bash
-    $ curl -skSL -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/mesos/master/slaves | jq '.slaves[] | .reserved_resources' | grep slave_public | wc -l
-    ```
-
+    
+        $ sudo -i /opt/mesosphere/bin/pkgpanda uninstall
+        $ sudo systemctl stop dcos-mesos-slave
+        $ sudo systemctl disable dcos-mesos-slave
+        
+
+2.  Remove the old directory structures on the agent node.
+    
+        $ sudo rm -rf /opt/mesosphere /var/lib/mesos
+        
+
+3.  Restart the machine.
+    
+        $sudo reboot
+        
+
+### Install DC/OS and convert to a public agent node
+
+1.  Copy the `dcos-install.tar` file to the node that that is being converted to a public agent. This `.tar` file is created during the GUI or CLI installation method.
+    
+        $ ssh $USER@$AGENT "sudo mkdir -p /opt/dcos_install_tmp"
+        $ scp ~/dcos-install.tar $ROOT_USER@$AGENT:~/dcos-install.tar
+        
+
+2.  SSH to the machine:
+    
+        $ ssh $USER@$AGENT
+        
+
+3.  Install DC/OS as a public agent:
+    
+        $ sudo mv ~/dcos-install.tar /opt/dcos_install_tmp
+        $ sudo bash /opt/dcos_install_tmp/dcos_install.sh slave_public
```

Comment:
As above, `tar cf ...` is added to the GUI and CLI modes; we don't compress the file because 98% of contents by volume are already compressed so there's no point and we actually save several seconds.
I think we should suggest that users capture and archive a copy of their install bits when they're done as just a matter of best practice even when using the GUI/CLI method, but I defer to your opinion @pyronicide and @joel-hamill.